packages/editor/src/components/post-author/constants.js

```diff
@@ -5,6 +5,6 @@ export const BASE_QUERY = {
 
 export const AUTHORS_QUERY = {
 	who: 'authors',
-	per_page: 50,
+	per_page: 100,
```

Comment:
I don't see any reason to limit this to 50, I just put 100 which I believe is the max.

Reply:
I don't remember the exact reason; it was probably just an arbitrary number we chose. I'll have a better look tomorrow.

Reply:
Confirmed. The `50` was just an arbitrary number. It might affect initial loading for the post editor on sites with a large user sets, but it's hard to measure locally.

Reply:
Happy to revert that if you think it's better for now, it's kind of outside the scope of the current PR.

Reply:
We can keep it. The query only fetches the `id,name` fields; the response size difference between 50 and 100 should be minimal.